pf_driver/cfg/PFDriverR2000.cfg

```diff
@@ -18,8 +18,8 @@ gen_r2000.add("subnet_mask",          str_t,    3, "IP netmask", "255.0.0.0")
 gen_r2000.add("gateway",              str_t,    4, "IP gateway", "0.0.0.0")
 gen_r2000.add("scan_frequency",       int_t,    5, "The parameter scan_frequency defines the set point for the rotational speed of the sensor head and therefore the number of scans recorded per second. For the R2000 valid values range from 10 Hz to 50 Hz with steps of 1 Hz.", 35, 10, 50)
 
-scan_direction_enum = gen_r2000.enum([ gen_r2000.const("cw",  int_t, 0, "Clocl-wise"),
-                                       gen_r2000.const("ccw", int_t, 1, "Counter clock-wise")],
+scan_direction_enum = gen_r2000.enum([ gen_r2000.const("cw",  str_t, "cw", "Clocl-wise"),
```

Comment:
"Clock-wise"

Reply:
Good catch! I'll add this when I do the other change.